### PR TITLE
Instant boosting

### PR DESCRIPTION
**Pull request by aminlatifi**
Implement instant boosting on backend side
https://github.com/Giveth/giveth-dapps-v2/issues/2571

### CODE REVIEW

**Comment by aminlatifi** [comment]
@CarlosQ96 @mohammadranjbarz Please note, I disable db sync for all ENVs except test

File: src/repositories/instantBoostingRepository.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import { AppDataSource } from '../orm';
+import { InstantPowerFetchState } from '../entities/instantPowerFetchState';
+import { BlockInfo } from '../adapters/givpowerSubgraph/IGivPowerSubgraphAdapter';
+
+// export const getLastInstantPowerUpdatedAt = async (): Promise<Number> => {
+//   const result = await InstantPowerBalance.createQueryBuilder('balance')
+//     .select('MAX(balance.chainUpdatedAt)', 'max')
+//     .getRawOne();
+//   return result?.max || 0;
+// };
```
Comment (CarlosQ96) [inline src/repositories/instantBoostingRepository.ts]: remove unused code if posible or we will use this later.

**Review by CarlosQ96 — APPROVED** [review]
Hey Amin I check the code looks good to me, great work.
I tested and check all files.

File: src/orm.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export class AppDataSource {
   static async initialize() {
     if (!AppDataSource.datasource) {
       const dropSchema = config.get('DROP_DATABASE') === 'true';
+      const synchronize = (config.get('ENVIRONMENT') as string) === 'test';
```
Comment (CarlosQ96) [inline src/orm.ts:13]: Just gotta remember from now on we have to write a migration for a new entity not just write the .ts file.

**Pull request merged**